alby.go

```diff
@@ -659,7 +659,7 @@ func (svc *AlbyOAuthService) CallbackHandler(c echo.Context) error {
 	}
 	client := svc.oauthConf.Client(c.Request().Context(), tok)
 
-	req, err := http.NewRequest("GET", fmt.Sprintf("%s/user/me", svc.cfg.AlbyAPIURL), nil)
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/internal/users", svc.cfg.AlbyAPIURL), nil)
```

Comment:
⚠️ Have to check if the `ALBY_CLIENT_SECRET` is the default in deployment repo

Reply:
what do you mean?

Reply:
We can only make calls to the internal api only using the internal client right?